### PR TITLE
seeAdmin

### PR DESCRIPTION
**Pull request by lxc150896**
redmine
https://dev.framgia.com/issues/210038
https://dev.framgia.com/issues/209809
https://dev.framgia.com/issues/209774
https://dev.framgia.com/issues/209472
task
https://dev.framgia.com/issues/208293

### CODE REVIEW

File: resources/assets/user/js/main.js [hunk 1]
```diff
@@ -438,16 +438,16 @@
...
         var e = $(event.target).attr('href');
-        var id = $(e).attr('id');
+        var status = $(e).attr('id');
+        var query = $('#userId').val;
         if ($(e).attr('value') == 'true') {
             $.ajax({
-                url: '/my-profile/' + id,
-                type: 'POST',
+                url: '/my-profile/' + status + '/' + $('#userId').val(),
+                method:"POST",
```
Comment (BunnyPi04) [inline resources/assets/user/js/main.js]: convention

File: resources/assets/user/js/main.js [hunk 2]
```diff
@@ -672,3 +672,8 @@
     });
 
 })(jQuery);
+
+    submitForms = function() {
+        document.forms["header-search"].submit();
+        return true;
```
Comment (BunnyPi04) [inline resources/assets/user/js/main.js:679]: thiếu dòng trắng trước return

Comment (BunnyPi04) [reply resources/assets/user/js/main.js:679]: convention dòng trên nữa

File: resources/assets/user/js/main.js [hunk 3]
```diff
@@ -438,16 +438,16 @@
...
         var e = $(event.target).attr('href');
-        var id = $(e).attr('id');
+        var status = $(e).attr('id');
+        var query = $('#userId').val;
         if ($(e).attr('value') == 'true') {
             $.ajax({
-                url: '/my-profile/' + id,
-                type: 'POST',
+                url: '/my-profile/' + status + '/' + $('#userId').val(),
+                method:'POST',
```
Comment (nguyenngocsond) [inline resources/assets/user/js/main.js:446]: để type cũng được mà

**Pull request merged**